modules/oe_translation_poetry/tests/Kernel/ContactFieldsTest.php

```diff
@@ -0,0 +1,191 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_translation_poetry\Kernel;
+
+use Drupal\field\Entity\FieldConfig;
+use Drupal\field\Entity\FieldStorageConfig;
+use Drupal\Tests\oe_translation\Kernel\TranslationKernelTestBase;
+
+/**
+ * Tests the Poetry contact fields type and formatter.
+ */
+class ContactFieldsTest extends TranslationKernelTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'oe_translation_poetry_html_formatter',
+    'oe_translation_poetry',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp() {
+    parent::setUp();
+
+    $this->installSchema('node', ['node_access']);
+
+    $this->container->get('entity_type.manager')->getStorage('node_type')->create([
+      'type' => 'page',
+      'name' => 'Page',
+    ])->save();
+
+    FieldStorageConfig::create([
```

Comment:
We could create a FieldManagementTrait to create fields and avoid duplicating code here and on PoetryRequestIdFieldTest.

Reply:
Even better we can check if similar traits already exist.

Reply:
Yes, I wanted to ask @22Alexandra to do this, then I took over and forgot :))

Reply:
In the end, I don't think there is much we can save..it's simple field creation.